src/transformState.js

```diff
@@ -5,7 +5,28 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const { type, extraData, keysToRemove } of actions) {
+    switch (type) {
+      case 'addProperties':
+        for (const [key, value] of Object.entries(extraData)) {
+          state[key] = value;
+        };
+        break;
+      case 'removeProperties':
+        for (const key of Object.values(keysToRemove)) {
+          if (key in state) {
+            delete state[key];
+          }
+        }
+        break;
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+        break;
+      default:
+    }
```

Comment:
empty default case makes no sense. Handle the case when the action type is wrong. Inform the user - console.log or throw an Error